core/src/main/java/org/apache/accumulo/core/client/Connector.java

```diff
@@ -128,6 +151,19 @@ public abstract BatchDeleter createBatchDeleter(String tableName, Authorizations
 
   public abstract BatchWriter createBatchWriter(String tableName, BatchWriterConfig config) throws TableNotFoundException;
```

Comment:
Could document the new merge behavior in javadocs.

Reply:
Added in ce512a93bfa9